### ISSUE TITLE
Can't install golint from current master (f8aef8e)

### ISSUE THREAD

**Issue by ggilder** [golang/lint#99]
I get the following error under Go 1.4, Mac OS X.

```
$ go install
# github.com/golang/lint
./lint.go:1432: assignment count mismatch: 2 = 3
```

**Comment by dsymonds** [comment]
Update your go.tools checkout. It is probably stale.

`go get -u golang.org/x/tools`

**Comment by ggilder** [comment]
Right you are, thank you!

**Comment by zimmski** [comment]
@dsymonds: I feel this issue is reported every week. Do you see a way to automate the update of /x/tools?

**Comment by dsymonds** [comment]
Unfortunately I don't think there is. This is an artifact of how the `go` tool works; if a package's API changes and a dependant package changes accordingly then you'll need to either have the old version of both or the new version of both.

**Comment by dsymonds** [comment]
I've added a `CONTRIBUTING.md` file to this repo to hopefully head people off.
